Lib/test/test_dataclasses.py

```diff
@@ -2883,6 +2883,11 @@ class C:
                 # x is not an InitVar, so there will be a member x.
                 self.assertEqual(C(10).x, 10)
 
+    def test_initvar_repr(self):
+        self.assertEqual(repr(InitVar[str]), 'dataclasses.InitVar[str]')
+        self.assertEqual(repr(InitVar[Optional[str]]),
```

Comment:
I am not sure that `InitVar[Optional[...]]` makes sense. The example in the documentation does not use `Optional`, but the default value is `None`. I suggest to use different example, e.g. `List[int]`

Reply:
I think optional does make sense, but I can change it if you like.